core/sql/udrserv/udrserv.cpp

```diff
@@ -170,24 +170,28 @@ static Int32 invokeUdrMethod(const char *method,
 // LCOV_EXCL_START
 // Dead Code
 // These methods are not used, and the interface has not been tested for a long time.
-// Andy is of the opinion that we might want to retire them
+// We might want to retire them
 static Int32 processCommandsFromFile(const char *filename, UdrGlobals &glob);
 static Int32 processSingleCommandFromFile(FILE *f, UdrGlobals &glob);
 // LCOV_EXCL_STOP
 
+// Changed the default to 512 to limit java heap size used by SQL processes.
+// Keep this define in sync with executor/JavaObjectInterface.cpp
+#define DEFAULT_JVM_MAX_HEAP_SIZE 512
```

Comment:
There are only a few shared header files between udr process and executor. Udr does not have the cli. Rather than put this in a basic header file like Platform.h or NAMemory.h I thought this would be better.